### ISSUE TITLE
Overwrites files if it renames two the same

### ISSUE THREAD

**Issue by christill** [christill/grunt-sanitize#2]
For example, if there are two files named ```my logo .jpg``` and ```my logo.jpg``` it will rename them both to ```my_logo.jpg``` causing it to overwrite and delete one.

**Comment by sabhiram** [comment]
Do you want to expose options to append a pattern on duplicate?

Like do you think we should rename things to `my_logo.jpg` and `my_logo_1.jpg`? Etc?

**Comment by christill** [comment]
Yeah that'd be the ideal. My plan was to just append and increment a number to the file name like you mentioned. What do you think?

**Comment by sabhiram** [comment]
Yea that is probably a decent first stab. I was also thinking about writing a few tests to validate these cases.

I noticed that the fixtures for the test is just a hardcoded file. Are you planning on expanding the flavors of tests? Typically I setup and teardown each test by creating a bunch of files to be sanitized etc. Thoughts?

**Comment by christill** [comment]
Yeah, that sounds better. This was my first time with testing in node. Feel free to alter as you see fit.
